PlayBook/src/getting_started/troubleshoot_login.md

```diff
@@ -1,40 +1,48 @@
+    - After disabling SIP, log back in to your macOS account
+    - Press `command ⌘` + `space bar` to open Spotlight Search and type in `terminal`
+    - Open the Terminal app, then copy and paste the following:
         - `sudo nvram boot-args="amfi_get_out_of_my_way=0x1 ipc_control_port_options=0"`
-    - If it appears that nothing has happened, this is correct.
+    - If prompted, authenticate with your password 
+    - If it appears that nothing has happened, this is normal
     - Restart your Mac
 
-3. Login to Genshin
-    - Open Genshin Impact with PlayCover, and you should be greeted with a Login button
-    - Login to your account, then wait until the door appears and quit the game with `Command (⌘)-Q`. **DO NOT CLICK/ENTER THE DOOR.**
+3. Login to Genshin Impact
+    - Open PlayCover
+    - Open Genshin Impact and you should be greeted with a login screen
+    - Log in to your account and complete the captcha as required
+    - **DO NOT ENTER WHEN THE DOOR APPEARS**[^1]
+    - Close the game using `command ⌘` + `Q` to avoid accidentally entering the door
 
-4. Enable SIP
+4. Re-enable SIP
     - Follow the steps in Step 1 to re-enter startup options
     - When `Utilities` appears in the menu bar, click on it and choose `Terminal`
     - In the terminal window type `csrutil enable` and type your password when prompted
-    - Once `Successfully enabled System Integrity Protection` appears, restart your Mac[^note]
+    - Wait for `Successfully enabled System Integrity Protection` to appear, then restart your Mac[^2]
 
-[^note]: The changes we made with the `nvram boot-args` command earlier will be disabled when SIP is on, however macOS will remember that we made those changes and will apply them the next time you disable SIP. Fully reverting these changes isn't neccassary for most users, but you can do so by running `sudo nvram boot-args=""` in a macOS terminal. 
-
-5. Open Genshin
+5. Open Genshin Impact
     - You're done! Enjoy playing Genshin!
 
-If you've already tried this and it did not work, try `sudo nvram boot-args="amfi_get_out_of_my_way=1"` as boot-args instead of the ones above. Also try deleting any items related to your game in Keychain Access and try enabling jailbreak detection bypass in your app settings. If your app still does not work, please open an app/game support request [here](https://github.com/PlayCover/PlayCover/issues/new/choose) after searching for similar issues.
+If you have tried this and still cannot login, try `sudo nvram boot-args="amfi_get_out_of_my_way=1"` as boot-args instead of the ones above. You can also try deleting any items related to your game in Keychain Access (type `keychain access` in Spotlight Search to open it), as well as try enabling jailbreak detection bypass in your app settings. If your app still does not work, you can [open an app/game support request](https://github.com/PlayCover/PlayCover/issues/new/choose) if no one else has done so already.
```

Comment:
```suggestion
If you have tried this and still cannot login, try `sudo nvram boot-args="amfi_get_out_of_my_way=1"` as boot-args instead of the ones above. You can also try deleting any items related to your game in Keychain Access (type `keychain access` in Spotlight Search to open it), as well as enabling jailbreak detection bypass in your app settings. Additional login issues may be solved by clearing app preferences.

If your app still does not work, you can [open an app/game support request](https://github.com/PlayCover/PlayCover/issues/new/choose) if no one else has done so already.
```